### PR TITLE
Add home page

### PR DESCRIPTION
**Pull request by Mukumbuta**
### Features implemented
- Added home page.
- Fetched movies from the `tvshows`  API using `async/await`.
- Rendered the movie list to the UI.
- Used `import/export`  syntax to import and export modules and images.
- Added a like icon to enable users to like the movies.
- Added styles to the home page.
- Fixed linter errors

### CODE REVIEW

File: src/style.scss [hunk 1]
```diff
@@ -56,6 +56,45 @@ header {
...
+    cursor: pointer;
+  }
+
+  .movies-likes {
+    margin-top: 15px;
+    margin-right: 50px;
+  }
+}
+
 .footer-section {
```
Comment (maaloum) [inline src/style.scss:98]: Kindly, Consider to adjust the margin-top of the footer

File: src/modules/requestAPIMovies.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import renderMovies from './renderMovies.js';
+
+const getAPIMovies = async () => {
+  const apiURL = 'https://api.tvmaze.com/shows';
+  const request = new Request(apiURL);
+  const response = await fetch(request);
+  const textMovieData = await response.text();
```
Comment (maaloum) [inline src/modules/requestAPIMovies.js:7]: Kindly, fetching data in JSON file format makes it easier to use with JavaScript

File: src/modules/renderMovies.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+                                            <div class="col movies-likes">
+                                                <img src="${likeIcon}" class="like-icon" alt="Red Heart Icon">
+                                                <p class="likes-holder">${movie.rating.average}</p>
+                                            </div> 
+                                        </div>
+                                    </div>`;
+  });
+};
+
+export default { renderMovies };
```
Comment (maaloum) [inline src/modules/renderMovies.js:24]: Since you have only one export module you can get rid of the curly brackets.

**Review by maaloum — COMMENTED** [review]
Hi @Mukumbuta 

Good job so far! I see you put a lot of effort into this task. However, there are some small issues that you still need to work on to enhance the home page but you are almost there!

## Highlights
- No linter errors
- Good documentation
- Good commit messages
- Good Git Workflow


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mukumbuta** [comment]
Hi @maaloum. Thank you for the review. I have since implemented all the changes you required.

**Review by maaloum — COMMENTED** [review]
@Mukumbuta 

Great job!!

Highlights!
Good feature branch name ✔️
No linter errors ✔️
Descriptive pull request ✔️
Good code organization and working app ✔️

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Status: Approved ✔️
Cheers and Happy coding!👏👏👏

Optional suggestions

Please ensure your PR description mentions an exact feature that you implemented. For example, you could say, "Updated webpack config file" or "I flagged the webpack script with --watch to keep watch of the changes in real-time", instead of "I set up webpack configuration." This may be confusing because earlier PRs show that another collaborator has already set up the webpack configuration file.
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by Mukumbuta** [comment]
Hi @maaloum. Thank you for reviewing and approving my PR

**Pull request merged**